frontend/test/playwright/e2e/load-more.spec.ts

```diff
@@ -158,4 +158,34 @@ test.describe("Load more button", () => {
       })
     })
   }
+
+  /**
+   * Checks that an analytics event is posted to /api/event and has the correct
+   * payload for the LOAD_MORE_RESULTS event.
+   */
+  test(`Sends a valid analytics event when loading more results`, async ({
```

Comment:
Did we want to have a test for pages beyond 1, or a test where the load more button is visible but not clicked and assert an event is not fired?

Reply:
ask and ye shall recieve 😄 good idea, added in befd63fcd